WolvenKit.Modkit/RED4/Tools/MeshImportTools.cs

```diff
@@ -551,7 +551,7 @@ private RawMeshContainer GltfMeshToRawContainer(Mesh mesh, GltfImportArgs args)
 
             if (mesh.Name != node.Name)
             {
-                _loggerService.Warning($"Mesh name `{mesh.Name}` does not match parent node name `{node.Name}`. Using {(args.OverrideMeshNameWithNodeName ? "NODE" : "MESH")} name.");
+                _loggerService.Info($"Mesh name `{mesh.Name}` does not match parent node name `{node.Name}`. Using {(args.OverrideMeshNameWithNodeName ? "NODE" : "MESH")} name.");
```

Comment:
This should remain a `Warning`, it's for a reason.

Reply:
OK, I'll put it back :)